code/datums/traits/negative.dm

```diff
@@ -478,3 +478,22 @@
 	..()
 	quirk_holder.update_client_colour() //Handle the activation of the colourblindness on the mob.
 	quirk_holder.update_icons() //Apply eyeshine as needed.
+
+/datum/quirk/mute/
+	name = "Mute"
+	desc = "You do not have a lot to say."
+	gain_text = "<span class='notice'>You become quiet.</span>"
+	lose_text = "<span class='notice'>You feel the need to express yourself via your Voice.</span>"
```

Comment:
I don't know, I feel both the gain and lose text should resemble more of their ability to produce sound rather than their willingness to. The `medical_record_text` already follows that in my opinion. That's unless this is just some mental block rather than a physical one but I don't know

Reply:
On gain:
`You don't feel like talking to people.`
and lose:

Reply:
Good point, I didn't think about that. I'll take a better look and adjust the critique